### DISCUSSION TITLE
Getting inaccurate results for solving natural convection in square thermal cavity for higher Rayleigh numbers f

### DISCUSSION THREAD

**Discussion by LeeStr1** [lululxvi/deepxde#1888] [Q&A]
I am trying to solve natural convection inside a square thermal cavity with deepXDE script. With lower rayleigh numbers such as 10^3 and 10^4, it is working perfectly fine and matching with the available validation data. 
But, with higher reynolds number, the solution is becoming too inaccurate. With quick google search and taking help of chatgpt, it suggest to do several things like increasing domain points, boundary points, increasing neural network layers and number of neurons etc. But, all these things are exponentially increasing the computational time (I don't have GPU, but only have CPUs). 

Also, somewhere I read, we should first train with lower reynolds number like 10^4 and then use the weights of loss functions, for further training with 10^5 (I am not sure how to do that). 

Can anyone please tell me what to do? what are the most critical things among these or some alternate way (which I didn't list here), which I must implement to achieve the accuracy for higher rayleigh number? 

Here is my current neural network architecture that is not working:

`data = dde.data.PDE(
    geom,
    pde,
    bcs,
    num_domain=4000,
    num_boundary=800,
    solution=None,
    num_test=600
)

net = dde.nn.FNN([2] + [100] * 6 + [4], "SILU", "Glorot normal")
model = dde.Model(data, net)

model.compile("adam", lr=0.001)
model.train(epochs=50000)
model.compile("L-BFGS")
model.train()`

**Comment by lululxvi** [comment]
Without GPU is hard.
